assets/src/edit-story/app/media/media3p/api/apiFetcher.js

```diff
@@ -162,14 +170,41 @@ class ApiFetcher {
     });
   }
 
-  async fetch({ params, path }) {
+  /**
+   * Perform a POST request to the Media3P API to register a usage.
+   * If the parameters are invalid or the server does not return a 200 response,
+   * an error is thrown.
+   *
+   * @param {Object} obj - An object with the options for the request.
+   * @param {string} obj.payload Payload to be posted to register media usage.
+   * @return {Promise<Object>} The response from the API.
+   */
+  async registerUsage({ payload }) {
+    validateRegisterUsagePayload(payload);
+
+    const params = [
+      ['payload', payload],
+      ['key', API_KEY],
+    ].filter((entry) => Boolean(entry[1]));
```

Comment:
The filter is not required, as none of the arguments are optional.

Reply:
I changes that a bit.